README.md

```diff
@@ -1,6 +1,6 @@
 # WorkOS Go Library
```

Comment:
are you planning to bump the version in separate PR? it's here https://github.com/workos/workos-go/blob/main/internal/workos/workos.go

Reply:
Thanks, meant to bump this as well